src/prometheus_alert_rules/juju_machines_unstable.rule

```diff
@@ -0,0 +1,8 @@
+alert: JujuMachinesUnstable
+expr: abs(sum(juju_machine_state) - sum(juju_machine_state offset 3m)) >= bool 0.33 * sum(juju_machine_state)
```

Comment:
Just a question what does `bool` do here ?

Reply:
It converted the result to boolean ie. if the expression results in some value, it returns 1, else 0. 
I removed it since in later tests I found out that 0 may also cause the alerts to be triggered.